modules/backend/ServiceProvider.php

```diff
@@ -23,12 +22,15 @@ public function register()
         $this->registerConsole();
         $this->registerMailer();
         $this->registerAssetBundles();
-        $this->registerBackendPermissions();
+
+        if (!$this->app->runningUnitTests()) {
+            $this->registerBackendPermissions();
+        }
```

Comment:
I think it would be preferable to fix the tests if possible. We don't want to end up with a situation where the tests operate differently to the actual functionality.

Reply:
The tests already operated differently from the core given that this call used to be wrapped in the runningInBackend() check instead.